sopel/coretasks.py

```diff
@@ -18,6 +18,7 @@
 import time
 import sopel
 import sopel.module
+import sopel.web
```

Comment:
It's unnecessary to import `sopel.submodule` if `sopel` is already imported, unless doing `from sopel.submodule import SpecificThing`.

Reply:
I was keeping the style of import used in `coretasks.py`. I don't like it. I'd rather have `from sopel import module, web, [...]`... but I would also keep that for another PR, if you agree.

Reply:
> another PR

Yeah, OK, I'll lay off for now and you can fix a bunch of those all at once. :+1: